docs/generate_docs_for_linux

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+doxygen Doxyfile.linux
```

Comment:
ditto, why having two platform-files, when all it does in run a python script (which is available on all platforms). Something like python -m doxygen .... should work here?
@breznak

Reply:
I have now moved the document generation documentation into docs/README.md
Doxyfile.common: I have specified the INPUT as a set of directories, not a set of files. If one wants to exclude certain files then one will have to specify the files in that directory explicitly.
As regards doxypypy, nothing has happened in https://github.com/Feneric/doxypypy for over a year. Maybe we should look at using something different. But this is at least a start. Without the patched version one can not use Python version > 3.7 neither on Linux nor on Windows.
The FILTER_PATTERNS are not the same for Linux and Windows, and the script files are not identical either. But I will try to merge them.

Reply:
I fail to see how I can merge the Doxyfiles and filter scripts. The syntax is different in bash versus Windows CMD: Argument specifier is $1 in bash but %1 in CMD.
It is possible to merge the generate_docs_ scripts, but the code becomes contrived:
  https://stackoverflow.com/questions/17510688/single-script-to-run-in-both-windows-batch-and-linux-bash